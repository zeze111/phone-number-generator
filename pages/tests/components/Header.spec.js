import React from "react";
import { shallow } from "enzyme";
import Header from "../../components/header";

describe("Header", () => {
  it("should render the component", () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
  });
});
