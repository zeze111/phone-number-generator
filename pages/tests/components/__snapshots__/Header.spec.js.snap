// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render the component 1`] = `
<header>
  <ul
    className="nav"
  >
    <li
      className="nav-link"
    >
      <a
        className="nav-list"
        href=""
      >
         TELECOMMS
      </a>
    </li>
    <li
      className="nav-link2 profile"
    >
      <a
        className="nav-list3"
        href=""
      >
         Admin
      </a>
    </li>
  </ul>
</header>
`;
